packages/devtools_app/lib/src/app.dart

```diff
@@ -443,6 +443,15 @@ class DevToolsScreen<C extends Object?> {
   ///
   /// If [createController] and [controller] are both null, [screen] will be
   /// responsible for creating and maintaining its own controller.
+  ///
+  /// The controller should expect to be already connected to an application
+  /// at the time of creation.
```

Comment:
This is not true. A controller does not require a connected application.

Reply:
rephrased